solutions/src/test/java/ru/sberbank/school/task08/JAXBManagerTest.java

```diff
@@ -0,0 +1,57 @@
+package ru.sberbank.school.task08;
+
+import org.junit.jupiter.api.*;
+import ru.sberbank.school.task08.state.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class JAXBManagerTest {
```

Comment:
Abbreviation in name &#39;JAXBManagerTest&#39; must contain no more than &#39;2&#39; consecutive capital letters.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1910729778>*